src/client/common/configSettings.ts

```diff
@@ -705,3 +718,23 @@ function isValidPythonPath(pythonPath: string): boolean {
         return false;
     }
 }
+
+function convertSettingTypeToLogLevel(setting: LoggingLevelSettingType): LogLevel {
+    switch (setting) {
+        case 'info': {
+            return LogLevel.Info;
+        }
+        case 'warn': {
+            return LogLevel.Warn;
+        }
+        case 'error': {
+            return LogLevel.Error;
+        }
+        case 'debug': {
+            return LogLevel.Debug;
+        }
+        default: {
+            return LogLevel.Off;
```

Comment:
Let's have this match the default in package.json:

```suggestion
        case 'off': {
            return LogLevel.Off;
        }
        default: {
            return LogLevel.Error;
```